frame/bounties/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "pallet-bounties"
+version = "2.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "Apache-2.0"
+homepage = "https://substrate.dev"
+repository = "https://github.com/paritytech/substrate/"
+description = "FRAME pallet to manage bounties"
+readme = "README.md"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+serde = { version = "1.0.101", optional = true, features = ["derive"] }
+codec = { package = "parity-scale-codec", version = "1.3.1", default-features = false, features = ["derive"] }
+sp-std = { version = "2.0.0", default-features = false, path = "../../primitives/std" }
+sp-runtime = { version = "2.0.0", default-features = false, path = "../../primitives/runtime" }
+frame-support = { version = "2.0.0", default-features = false, path = "../support" }
+frame-system = { version = "2.0.0", default-features = false, path = "../system" }
+pallet-balances = { version = "2.0.0", default-features = false, path = "../balances" }
```

Comment:
This shouldn't be a hard dependency I think. Move it to `dev-dependencies`

Reply:
Good one, Indeed it should be part of dev-dependencies, and moved "pallet-balances" as part of dev-dependencies